tljh/schemas/tljh_configurator.py

```diff
@@ -0,0 +1,27 @@
+from jupyterhub_configurator.hooks import hookimpl
```

Comment:
Let's put this in just tljh/configurator_schemas.py, instead of making its own modules? In the future, we might want to use Jupyter Telemetry for something, and that will have its own schemas.